package.json

```diff
@@ -774,6 +785,21 @@
         "view": "github:compareChanges",
         "when": "github:noUpstream",
         "contents": "The comparing branch has no upstream remote\n[Publish branch](command:git.publish)"
+      },
+      {
+        "view": "pr:github",
+        "when": "!github:initialized",
+        "contents": "Loading pull requests..."
```

Comment:
Actually, there's a point later when we are actually loading pull requests where we display "Loading...", so I think this text might be better as "Initializing..."